src/AppBundle/Resources/views/Firm/show.json.twig

```diff
@@ -0,0 +1,36 @@
+            
+{
+
+   "@context": "http://schema.org",
+   "@type": "Organization",
+   
+   "url": "{{ url('firm_show', {'id': firm.id}) }}",
+   
+   "name": "{{ firm.name }}",
+   "address": "{{ firm.streetAddress }}",
+   
+   "location": {
+   "@type": "Place",
+   "url": ""http://sws.geonames.org/{{ firm.city.geonameid }}/about.rdf""
+   },
+
+   
+   {% if firm.startDate|date('Y') %}
+        "foundingDate": "{{ firm.startDate }}",
+        {% endif %}
+   
+   {% if firm.endDate|date('Y')  %}
+        "dissolutionDate": "{{ firm.endDate }}",
+        {% endif %}
+   
+   {% for tfr in firm.TitleFirmroles %}
```

Comment:
If a firm has multiple titleFirmRoles, only the first (or last, depending on the JSON processor) will get processed. They should be in a list.